src/@carbon-info/views/Ecosystem/Ecosystem.tsx

```diff
@@ -2,432 +2,506 @@ import { CTAButton, FadeAndSlide } from "@carbon-info/components";
 import { Path } from "@carbon-info/constants";
 import { useContentful } from "@carbon-info/hooks";
 import { StyleUtils } from "@carbon-info/utils/styles";
-import { Box, Button, Grid, Theme, Typography, makeStyles } from "@material-ui/core";
+import {
+	Box,
+	Button,
+	Grid,
+	Theme,
+	Typography,
+	makeStyles,
+} from "@material-ui/core";
 import clsx from "clsx";
 import React, { useEffect, useState } from "react";
 import { useInView } from "react-intersection-observer";
 import { AppsCarousel, FeatureGrid } from "./components";
-import { BlockchainConfig, ValidatorConfig, WalletConfig, DAppsConfig } from "./ecosystemConfig";
+import { ReactComponent as SWTHIcon } from "@carbon-info/assets/icons/SWTH.svg";
+import {
+	BlockchainConfig,
+	ValidatorConfig,
+	WalletConfig,
+	DAppsConfig,
+} from "./ecosystemConfig";
+import Banner from "@carbon-info/components/Banner";
 
 interface SubTextContent {
-  description: string
-  link: string
+	description: string;
+	link: string;
 }
 
 const Ecosystem: React.FC = () => {
-  const classes = useStyles();
-  const { ref, inView } = useInView({
-    /* Optional options */
-    threshold: 0.35,
-    triggerOnce: true,
-  });
-  const { data } = useContentful({
-    contentType: "carbonEcosystemEntry",
-  });
+	const classes = useStyles();
+	const { ref, inView } = useInView({
+		/* Optional options */
+		threshold: 0.35,
+		triggerOnce: true,
+	});
+	const { data } = useContentful({
+		contentType: "carbonEcosystemEntry",
+	});
 
-  const [allBlockchains, setAllBlockchains] = React.useState<BlockchainConfig[]>([]);
-  const [allWallets, setAllWallets] = React.useState<WalletConfig[]>([]);
-  const [allValidators, setAllValidators] = React.useState<ValidatorConfig[]>([]);
-  const [allDAppsEVM, setAllDAppsEVM] = React.useState<DAppsConfig[]>([]);
-  const [allDAppsCore, setAllDAppsCore] = React.useState<DAppsConfig[]>([]);
+	const [allBlockchains, setAllBlockchains] = React.useState<
+		BlockchainConfig[]
+	>([]);
+	const [allWallets, setAllWallets] = React.useState<WalletConfig[]>([]);
+	const [allValidators, setAllValidators] = React.useState<ValidatorConfig[]>(
+		[],
+	);
+	const [allDAppsEVM, setAllDAppsEVM] = React.useState<DAppsConfig[]>([]);
+	const [allDAppsCore, setAllDAppsCore] = React.useState<DAppsConfig[]>([]);
 
+	useEffect(() => {
+		async function fetchEcosystemItems() {
```

Comment:
There's no need to bind the data processing in an async function as the data should alr be available on JSON format. Pls remove the async function wrapping.

For a more detailed explanation, pls refer to comment on L45 of `src/@carbon-info/views/Data/Data.tsx`.